tests/tx/test_tx_storage.py

```diff
@@ -146,11 +146,11 @@ def test_storage_basic(self):
             self.assertEqual(2, self.tx_storage.get_tx_count())
             self.assertEqual(3, self.tx_storage.get_count_tx_blocks())
 
-            block_parents_hash = [x.data for x in self.tx_storage.get_block_tips()]
+            block_parents_hash = self.tx_storage.get_best_block_tips()
             self.assertEqual(1, len(block_parents_hash))
             self.assertEqual(block_parents_hash, [self.genesis_blocks[0].hash])
 
-            tx_parents_hash = [x.data for x in self.tx_storage.get_tx_tips()]
+            tx_parents_hash = self.manager.get_new_tx_parents()
```

Comment:
I feel this change is testing a different thing. Maybe we should test both `get_tx_tips()` and `get_new_tx_parents()`?

Reply:
Makes sense.

Reply:
So the old test is using `get_tx_tips()` while the new one is using `get_new_tx_parents()`. Great! Thanks!